packages/design-picker/src/__tests__/integration.test.tsx

```diff
@@ -1,6 +1,47 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { isEnabled } from '@automattic/calypso-config';
```

Comment:
(sorry I seem to have deleted your previous comment by accident!)

Have you tried to 

```suggestion
import '@automattic/calypso-config';
```

Similar to what I did in https://github.com/Automattic/wp-calypso/blob/trunk/packages/design-picker/src/utils/__tests__/available-designs.test.ts

Reply:
Thanks for the suggestion! I applied it in 22410b424d387345281ab336c817a9953191e4d9